<!--
Desc:
  Generates map->odom transform to correct for sensor degeneracy by shifting the degenerate
  odom->base transform to a non-degenerate base frame.
-->

<launch>
  <node name="degeneracy_corrector" pkg="degeneracy_corrector" type="corrector.py" output="screen" >
    <param name="map_frame"              value="map" />
    <param name="odom_target_topic"      value="/filtered/odom" />
    <param name="odom_degenerate_topic"  value="/sensor/odom" />
    <param name="bool_degeneracy_topic"  value="/sensor/is_degenerate" />

    <!-- OTHER -->
    <param name="update_rate"            value="10" />
  </node>
</launch>
